pkg/sql/show_tenant.go

```diff
@@ -32,6 +34,7 @@ import (
 type showTenantNode struct {
 	name               tree.TypedExpr
 	tenantInfo         *descpb.TenantInfo
+	replicationStatus  string
```

Comment:
nit: should we call this tenantState instead of replicationStatus given it can also be `ADD`, `DROP`, and `ACTIVE`?